contrib/hadoop/pom.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add AUTHORS.md and README.md files. Please follow the pattern established by the [`join-library`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/tree/master/contrib/join-library). The README should cover all documentation needed to use the artifact.